### PR TITLE
[Components with more than 1 service] LPS-192993 Refactor ManagedServiceFactory from oauth2-provider-rest, document-library-service, document-library-preview-pdf and portal-security-sso-openid-connect-impl modules

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-192993](https://liferay.atlassian.net/browse/LPS-192993)

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:security

**Comment by RenanUmV** [comment]
ci:test:document

File: modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/configuration/admin/service/PDFPreviewManagedServiceFactory.java [hunk 1]
```diff
@@ -182,251 +66,54 @@ public void updated(String pid, Dictionary<String, ?> dictionary)
...
-			_updateSystemPDFPreviewConfiguration(maxNumberOfPages);
-		}
-		else {
-			throw new PortalException("Unsupported scope: " + scope);
-		}
-	}
-
 	@Activate
 	@Modified
 	protected void activate(Map<String, Object> properties) {
```
Comment (rafaprax) [inline modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/configuration/admin/service/PDFPreviewManagedServiceFactory.java:71]: Hey @RenanUmV, move the activate to the helper class. please

File: modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/configuration/admin/service/PDFPreviewManagedServiceFactory.java [hunk 2]
```diff
@@ -41,9 +30,7 @@
 @Component(
 	configurationPid = "com.liferay.document.library.preview.pdf.internal.configuration.PDFPreviewConfiguration",
```
Comment (rafaprax) [inline modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/configuration/admin/service/PDFPreviewManagedServiceFactory.java:31]: Hey @RenanUmV, move the `configurationPid = "com.liferay.document.library.preview.pdf.internal.configuration.PDFPreviewConfiguration"` to the helper class as well. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
@RenanUmV Please resend when you apply the changes requested. Thx

**Pull request closed without merging**